README.md

```diff
@@ -20,59 +19,53 @@ var initTracer = require('jaeger-client').initTracer;
 var config = {
-  'serviceName': 'my-awesome-service'
+  serviceName: 'my-awesome-service',
 };
 var options = {
-  'tags': {
-    'my-awesome-service.version': '1.1.2'
+  tags: {
+    'my-awesome-service.version': '1.1.2',
   },
-  'metrics': metrics,
-  'logger': logger
+  metrics: metrics,
+  logger: logger,
 };
 var tracer = initTracer(config, options);
 ```
 
 ## Usage
 
-The Tracer instance created by `initTracer` is OpenTracing-1.0 compliant.
-See [opentracing-javascript](https://github.com/opentracing/opentracing-javascript) for usage examples.
+The Tracer instance created by `initTracer` is OpenTracing-1.0 compliant. See [opentracing-javascript](https://github.com/opentracing/opentracing-javascript) for usage examples.
 
 ### TChannel Span Bridging
 
-Because [tchannel-node](https://github.com/uber/tchannel-node) does not have instrumentation
-for OpenTracing, Jaeger-Client exposes methods wrapping tchannel handlers, and encoded channels.
-An encoded channel is a channel wrapped in either a thrift encoder `TChannelAsThrift`,
-or json encoder `TChannelAsJson`.  To wrap a server handler for thrift one can initialize
-a tchannel bridge, and wrap the encoded handler function with a `tracedHandler` decorator.
-The tchannel bridge takes an OpenTracing  tracer, and a context factory.  The context factory
-must be a function that returns a context with the methods 'getSpan', and 'setSpan' which retrieve
-and assign the span to the context respectively.
+Because [tchannel-node](https://github.com/uber/tchannel-node) does not have instrumentation for OpenTracing, Jaeger-Client exposes methods wrapping tchannel handlers, and encoded channels. An encoded channel is a channel wrapped in either a thrift encoder `TChannelAsThrift`, or json encoder `TChannelAsJson`. To wrap a server handler for thrift one can initialize a tchannel bridge, and wrap the encoded handler function with a `tracedHandler` decorator. The tchannel bridge takes an OpenTracing tracer, and a context factory. The context factory must be a function that returns a context with the methods 'getSpan', and 'setSpan' which retrieve and assign the span to the context respectively.
 
 ```javascript
 import { TChannelBridge } from 'jaeger-client';
 import Context from 'some-conformant-context';
 
 function contextFactory() {
-    return new Context();
-};
+  return new Context();
+}
 
-let bridge = new TChannelBridge(tracer, {contextFactory: contextFactory});
+let bridge = new TChannelBridge(tracer, { contextFactory: contextFactory });
 let server = new TChannel({ serviceName: 'server' });
 server.listen(4040, '127.0.0.1');
 let serverThriftChannel = TChannelAsThrift({
-    channel: server,
-    entryPoint: path.join(__dirname, 'thrift', 'echo.thrift') // file path to a thrift file
+  channel: server,
+  entryPoint: path.join(__dirname, 'thrift', 'echo.thrift'), // file path to a thrift file
```

Comment:
why comma at the end? I don't think JS allows that (unlike Go/Python)

Reply:
because [.prettierrc#L4](https://github.com/jaegertracing/jaeger-client-node/blob/master/.prettierrc#L4) (and yes JS allows it as well 😄)

Reply:
The trailing comma is find for object literals: [Trailing_commas#Objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#Objects).